backend/app.ts

```diff
@@ -1,13 +1,13 @@
-const express = require('express')
-const bodyParser = require('body-parser')
+import * as express from 'express'
```

Comment:
Just a theoretical question - we might want to replace this by importing just the objects that are being used 🤔 But that can of course be done in the future in a separate PR.

Reply:
I think for our modules it will be done in future PR. Regarding 3rd party - actually I realize now that it should be possible to import, for example, "express" without *. However I struggle with this and get errors about default import or that express is not a function :(

Reply:
I see from performance-review project that probably you meant smth like this:
```
import {
  Application, NextFunction, Request, Response,
} from 'express'

const express: Application = express()
```
Yes, I think it can be done other PR when we get lint rules for reporting it as a warning or error.

Reply:
it seems ok to import express with * or use require - Microsoft does it here with * https://github.com/microsoft/TypeScriptSamples/blob/master/imageboard/app.ts. So I guess I can merge then :)